app/views/users/index.html.erb

```diff
@@ -0,0 +1,15 @@
+<!-- <%= render "shared/navbar" %> -->
+
+<main class="main-page-container">
+  <p style="color: green"><%= notice %></p>
+
+  <h1>Welcome to the Buget Tracker App</h1>
+
+  <p>This Application will help you track all of the expenses you make in different cateogires, in order for you to have a better management of your economy</p>
+
+  <i class="fa-solid fa-magnifying-glass-dollar fa-6x"></i>
+</main>
+
+<footer>
+
```

Comment:
- [ ] The design of the splash screen should be similar to the given design. Please fix this.

### Given design
![Screenshot from 2022-10-30 00-28-17](https://user-images.githubusercontent.com/26767383/198850520-94e5e863-d8bd-4265-b8d2-06ce19c6d6cf.png)


### Your design
![Screenshot from 2022-10-30 01-28-26](https://user-images.githubusercontent.com/26767383/198850513-47c42c61-a879-4457-bdf6-96c4b08d73a2.png)